segmentation/model/sncn_resnet.py

```diff
@@ -0,0 +1,651 @@
+import torch
```

Comment:
I think this file is modified from torchvision's official resnet file, please add a header here to indicate the code source and license.